wpilibc/src/main/native/cpp/smartdashboard/SendableBuilderImpl.cpp

```diff
@@ -235,6 +292,22 @@ void SendableBuilderImpl::AddRawProperty(
   m_properties.emplace_back(std::move(prop));
 }
 
+void SendableBuilderImpl::PublishConstRaw(std::string_view key,
+                                          std::string_view typeString,
+                                          std::span<const uint8_t> value) {
+  auto topic = m_table->GetRawTopic(key);
+  auto prop = std::make_unique<PropertyImpl<nt::RawTopic>>();
+  prop->pub = topic.Publish(typeString);
+  prop->updateNetwork = [value, prevPub = NT_Publisher{}](
+                            auto& pub, int64_t time) mutable {
+    if (prevPub != pub.GetHandle()) {
+      pub.Set(value, time);
```

Comment:
This needs to be fixed to match the changes to PublishConstImpl (and probably can just call PublishConstImpl instead).

Reply:
It can't call `PublishConstImpl` because of the difference in parameter to `Publish()`.